Gems/AWSCore/Code/Tools/ResourceMappingTool/utils/json_utils.py

```diff
@@ -28,7 +28,7 @@
 
 RESOURCE_MAPPING_ACCOUNTID_JSON_KEY_NAME: str = "AccountId"
 RESOURCE_MAPPING_ACCOUNTID_TEMPLATE_VALUE: str = "EMPTY"
-_RESOURCE_MAPPING_ACCOUNTID_PATTERN: str = f"^[0-9]{{12}}|{RESOURCE_MAPPING_ACCOUNTID_TEMPLATE_VALUE}$"
+_RESOURCE_MAPPING_ACCOUNTID_PATTERN: str = f"^[0-9]{{12}}|{RESOURCE_MAPPING_ACCOUNTID_TEMPLATE_VALUE}|$"
```

Comment:
the regex after the [` | `] should be [ `^$` ] (not just `$`), as "`^`" denotes the first character:
https://stackoverflow.com/questions/3333461/regular-expression-which-matches-a-pattern-or-is-an-empty-string